datasources/nm.ejs

```diff
@@ -17,7 +17,7 @@
   ],
   "catalog": [
         {
-            "name": "National Data Sets",
+            "name": "National Datasets",
```

Comment:
This change will break all existing share URLs.  I _thought_ we could work around this by adding a `shareKeys` property, but it's not working and it looks like it might take quite a bit of work to make it work.  Would you be willing to skip the item renames for now?